CONTRIBUTING.md

```diff
@@ -46,5 +46,8 @@ docker-compose --project-name django-multitenant up
 Then in another shell run the tests:
 
 ```bash
+DJANGO_VERSION=4.0
+CITUS_VERSION=11.2
+make test-dependencies 
```

Comment:
Hm, didn't work for me because the make task uses an absolute path. Am I supposed to execute those commands in a Docker container, or right from my system shell?

```
pip install -r /build/requirements/test-requirements.txt
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not open requirements file: [Errno 2] No such file or directory: '/build/requirements/test-requirements.txt'
```

Reply:
You're right. I needed to change absolute path into relative. 
Could you check now?

Reply:
```
$ make test-dependencies
pip install -r requirements/test-requirements.txt
Defaulting to user installation because normal site-packages is not writeable
Collecting attrs==22.2.0
  Using cached attrs-22.2.0-py3-none-any.whl (60 kB)
ERROR: Could not find a version that satisfies the requirement coverage[toml]==7.0.5 (from versions: 3.0b3, 3.0, 3.0.1, 3.1b1, 3.1, 3.2b1, 3.2b2, 3.2b3, 3.2b4, 3.2, 3.3, 3.3.1, 3.4b1, 3.4b2, 3.4, 3.5b1, 3.5, 3.5.1b1, 3.5.1, 3.5.2b1, 3.5.2, 3.5.3, 3.6b1, 3.6b2, 3.6b3, 3.6, 3.7, 3.7.1, 4.0a1, 4.0a2, 4.0a3, 4.0a4, 4.0a5, 4.0a6, 4.0b1, 4.0b2, 4.0b3, 4.0, 4.0.1, 4.0.2, 4.0.3, 4.1b1, 4.1b2, 4.1b3, 4.1, 4.2b1, 4.2, 4.3, 4.3.1, 4.3.2, 4.3.3, 4.3.4, 4.4b1, 4.4, 4.4.1, 4.4.2, 4.5, 4.5.1, 4.5.2, 4.5.3, 4.5.4, 5.0a1, 5.0a2, 5.0a3, 5.0a4, 5.0a5, 5.0a6, 5.0a7, 5.0a8, 5.0b1, 5.0b2, 5.0, 5.0.1, 5.0.2, 5.0.3, 5.0.4, 5.1, 5.2, 5.2.1, 5.3, 5.3.1, 5.4, 5.5, 5.6b1, 6.0b1, 6.0, 6.0.1, 6.0.2, 6.1, 6.1.1, 6.1.2, 6.2)
ERROR: No matching distribution found for coverage[toml]==7.0.5
make: *** [test-dependencies] Error 1
```